Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Spawning/SpawnTicketInstance.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/RTTI/RTTI.h>
+#include <AzCore/RTTI/ReflectContext.h>
+#include <AzFramework/Spawnable/SpawnableEntitiesInterface.h>
+
+namespace ScriptCanvas::Nodeables::Spawning
+{
+    //! Wrapper around EntitySpawnTicket that can be used by ScriptCanvas
+    struct SpawnTicketInstance final
+    {
+        AZ_RTTI(SpawnTicketInstance, "{2B5EB938-8962-4A43-A97B-112F398C604B}");
+        static void Reflect(AZ::ReflectContext* context);
+
+        SpawnTicketInstance() = default;
+        SpawnTicketInstance(const SpawnTicketInstance& rhs);
+        SpawnTicketInstance& operator=(const SpawnTicketInstance& rhs);
```

Comment:
There doesn't seem to be a need to define the copy constructor/assignment operator. If anything, they can just be set to `= default`, although that would still prevent them from being moved. If they are removed than the default constructor and destructor can also be removed.
This may also apply to the other cases where a blank copy constructor and assignment operator were added.